main/FatesHistoryInterfaceMod.F90

```diff
@@ -3872,34 +3883,69 @@ subroutine update_history_dyn(this,nc,nsites,sites,bc_in)
 
       ! pass the cohort termination mortality as a flux to the history, and then reset the termination mortality buffer
       ! note there are various ways of reporting the total mortality, so pass to these as well
+      do i_pft = 1, numpft
+         hio_cstarvmortality_carbonflux_si_pft(io_si,i_pft) = hio_cstarvmortality_carbonflux_si_pft(io_si,i_pft) + &
+              (sites(s)%term_carbonflux_ustory(i_term_mort_type_cstarv,i_pft) + &
+              sites(s)%term_carbonflux_canopy(i_term_mort_type_cstarv,i_pft)) * days_per_sec * ha_per_m2
```

Comment:
Given this addition, should the entry for this variable at line 3400 be removed?  Or is this intended to sum on top of the previous calculation?

Reply:
I think it's intended to sum on top of the previous calculation but can you confirm this @ckoven ?

Reply:
yes it is in addition